modules/blocks.php

```diff
@@ -256,16 +255,20 @@ function jetpack_business_hours_render( $attributes, $content ) {
 	}
 
 	foreach ( $attributes['hours'] as $day => $hours ) {
-		$opening = strtotime( $hours['opening'] );
-		$closing = strtotime( $hours['closing'] );
-
-		$content .= '<dt class="' . esc_attr( $day ) . '">' .
-			ucfirst( $wp_locale->get_weekday( array_search( $day, $days ) ) ) .
-			'</dt>';
-		$content .= '<dd class="' . esc_attr( $day ) . '">';
-		if ( $hours['opening'] && $hours['closing'] ) {
-			$content .= sprintf(
-				/* Translators: Business opening hours info. */
+		$content    .= '<dt class="' . esc_attr( $day ) . '">' .
+					   ucfirst( $wp_locale->get_weekday( array_search( $day, $days ) ) ) .
+					   '</dt>';
+		$content    .= '<dd class="' . esc_attr( $day ) . '">';
+		$days_hours = '';
+
+		foreach ( $hours as $hour ) {
```

Comment:
Should we add a failsafe here to make sure that we can only have one set of opening-closing hours per day? I can get into situations like this right now:

![image](https://user-images.githubusercontent.com/426388/52477752-578c8000-2ba3-11e9-9998-b04064392e63.png)

(I am not quite sure how I got into that situation to be honest :) )